src/qml/qgismobileapp.qml

```diff
@@ -1332,7 +1332,7 @@ ApplicationWindow {
     }
 
     MenuItem {
-      text: qsTr( "View Log Messages" )
+      text: qsTr( "Messages Logs" )
```

Comment:
Compare with "Message Log" in the toasts. Keep it singular?

Reply:
Now this is displayed correctly